ibis/backends/tests/sql/snapshots/test_compiler/test_agg_filter/out.sql

```diff
@@ -1,10 +1,16 @@
 SELECT
-  t0.a,
-  t0.b * CAST(2 AS TINYINT) AS b2
-FROM my_table AS t0
+  t1.a,
+  t1.b2
+FROM (
+  SELECT
```

Comment:
These would nice to be squashed, is this change relevant to the rewrite change?

Reply:
Yep, I believe it is.